output-core/src/models/Context.ts

```diff
@@ -1,74 +1,22 @@
-import {
-  IsNotEmpty,
-  IsOptional,
-  IsString,
-  TransformMap,
-  Type,
-  ValidateNested,
-} from '@jovotech/output';
-
-export class Intent {
-  @IsString()
-  @IsNotEmpty()
-  name: string;
-}
-
-export class Input {
-  [key: string]: unknown;
+import { IsOptional, Type, ValidateNested } from '@jovotech/output';
+import { Session } from './Session';
+import { User } from './User';
 
+export class Request {
   @IsOptional()
-  @IsString()
-  @IsNotEmpty()
   id?: string;
```

Comment:
A validation decorator to check if it is a string is missing. Please add `@IsString()`